### ISSUE TITLE
Result with pre-trained model

### ISSUE THREAD

**Issue by vincentleon** [UnrealLink/TP-GAN#5]
Hi, thanks for making this model available !
I tried testing with the pre-trained model given for the test images in image_list_reduced.yml

![image](https://user-images.githubusercontent.com/3737387/63771844-38073b80-c8d8-11e9-9ebd-cdb8332d2133.png)
From what I understand, the second column is the generated result, and the last one is the ground truth.
Is there an error on my part, or is that result a work in progress ?

**Comment by eeewhe** [comment]
It looks like that the quality of generated images is bad.

**Comment by redaghanem** [comment]
Does anyone train this MODEL?

**Comment by UnrealLink** [comment]
Hi, sorry for the late response.
This implementation was done as part of a class project, with limited time (and skills). I haven't had the time yet to go back to it to do a proper implementation.
The results you obtained were the ones I had as well. I don't know if this is due to errors in the code base (my memories are a bit foggy but one of the loss function was wrongly implemented I believe), or the smaller dataset I had to use since MultiPIE (used in the article) was not freely available.
